_inc/client/at-a-glance/scan.jsx

```diff
@@ -154,11 +157,16 @@ class DashScan extends Component {
 			} );
 		}
 
+		const isRewindActive = includes(
+			[ 'active', 'provisioning', 'awaiting_credentials' ],
```

Comment:
`awaiting_credentials` is not active, so during this time the message 

> We are making sure your site stays free of security threats. You will be notified if we find one.

Is not true.

Reply:
I'm aware of it and didn't want to amend this at this stage because we will revisit this soon and also because we have the JITM prompting user to enter creds at the top. This simply fixed the inconsistency between having VP or Rewind.
However, I fully agree with the comment and yeah, we could update it, it's not too complex.
@MichaelArestad would a text with link to the flow to enter creds be enough? What should the text be?

Reply:
Yeah. Text with a link to the flow to enter creds would be good. We could use the Banner style and say the same thing as seen here:

<img width="1055" alt="image" src="https://user-images.githubusercontent.com/1123119/37976591-0a558b0a-319f-11e8-9183-bc4b6df1fe51.png">

Perhaps just say "Security scanning" instead of "Backups and security scans".